tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist =
+    py{27,35,36,37}
+    flake8
+skip_missing_interpreters = true
+toxworkdir = {toxinidir}/build/tox
+
+[testenv]
+install_command = pip install --upgrade --upgrade-strategy eager --find-links https://manthey.github.io/large_image_wheels {opts} .[memcached] {packages}
+deps = -rrequirements-dev.txt
+whitelist_externals = 
+    rm
+    npx
+commands =
+    rm -rf build/test/coverage/web_temp
+    girder build --dev
+    pytest --cov-config setup.cfg {posargs}
+    npx nyc report --temp-dir build/test/coverage/web_temp --report-dir build/test/coverage --reporter cobertura --reporter text-summary
```

Comment:
Why is npm install/testing being invoked by tox?

Reply:
This project contains end-to-end testing (i.e. tests that exercise both the server side and the web client simultaneously), so we require a test harness that spans both environments. Pytest was the chosen harness, and it was already being invoked by tox.

(Maybe I answered the wrong question... do you have an alternate proposal?)